### ISSUE TITLE
Support create without installing deps

### ISSUE THREAD

**Issue by zhaoyao91** [jaredpalmer/tsdx#685]
### Current Behavior

create tsdx package will always install the deps

### Desired Behavior

support a flag to prevent installing deps. so we could generate a project quickly to compare it's files.

### Suggested Solution

add a flag such as `--no-install`

### Additional context

tsdx: 0.13.2

**Comment by agilgur5** [comment]
I don't believe `create-react-app` has such a feature. But this could help with things like #419 .
